user/standard/passphrase.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+grand_parent: User Admin
+parent: Standard
+---
+
+# passphrase
+
+User administration functions for modifying a user's passphrase. 
+{: .fs-6 .fw-300 }
+
+## `UserAdmin.set_passphrase()`
+<br>
+
+<br>
+```python
+def set_passphrase(self, userid: str, passphrase: str) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+&nbsp;
+
+{: .warning }
+> * _pyRACF encodes the data it passes to RACF in Code Page `IBM-1047`._
+> * _If you are entering a passphrase with special or national characters, users logging on from terminals using differnt or international codepages may experience errors._
+> * _Please consult a list of invariant characters to use for such passphrases or phrases if this applies to you._
+
+&nbsp;
+
+{: .warning }
+> * _All occurances of the specified passphrase in the returned **Security Result Steps dictionary** or **Concatenated Security Request XML bytes** are redacted._
+> * _When the `debug` class attribute is `True`, all occurances of the specified passphrase will be redacted in debug messages produced by this function._
+
+&nbsp;
+
+Change a user's **passphrase**.
+
+#### 📥 Parameters
+* `userid`<br>
+  The **z/OS userid** of the user who's **passphrase** is being changed.
+
+* `passphrase`<br>
+  The **passphrase** to assigned to the specified user.
```

Comment:
Change `assigned` to `assign`. Same typo exists in `user/standard/password.md` as well.

Reply:
Nice catch, typo fixed in both places.